### DISCUSSION TITLE
webpack output multiple html file can't work

### DISCUSSION THREAD

**Discussion by ghost** [webpack/webpack#13150] [Q&A]
Hey guys, I m using webpack to output multiple html file.
But I don't know why it just output 1 html file.

my webpack.config.ts is like this.

`const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

const webpackConfig = {
    mode: 'development',

    entry: {
        'index': './src/index.tsx',
        'sub/save_list': './src/sub/save_list.tsx'
    },

    output: {
        path: path.resolve(__dirname, 'dist'),
        filename: '[name].js'

    },

    resolve: {
        extensions: ['.js', '.json', '.ts', '.tsx', 'css']
    },

    devtool: 'source-map',

    module: {
        rules: [
            {
                test: /\.tsx?$/,
                use: "ts-loader",
                include: path.resolve(__dirname, 'src'),
                exclude: /node_modules/
            }
        ]
    },

    plugins: [
        new HtmlWebpackPlugin({
            template: './src/index.html',
            __filename: 'index.html',
            chunks: ['index'],
        }),
        new HtmlWebpackPlugin({
            template: './src/sub/save_list.html',
            __filename: 'save_list.html',
            chunks: ['sub/save_list'],
        })
    ]
};
module.exports = webpackConfig`

when I use  "npm run build" , the result is this.

`react_ts@1.0.0 build
 webpack

Version: webpack 4.46.0
Time: 3856ms
Built at: 2021-04-15 18:10:30
               Asset       Size         Chunks                   Chunk Names
          index.html  395 bytes                 [emitted]
            index.js   1010 KiB          index  [emitted]        index
        index.js.map   1.15 MiB          index  [emitted] [dev]  index
    sub/save_list.js    949 KiB  sub/save_list  [emitted]        sub/save_list
sub/save_list.js.map   1.08 MiB  sub/save_list  [emitted] [dev]  sub/save_list
Entrypoint index = index.js index.js.map
Entrypoint sub/save_list = sub/save_list.js sub/save_list.js.map
[./src/app.tsx] 3.89 KiB {index} [built]
[./src/index.tsx] 497 bytes {index} [built]
[./src/sub/save_app.tsx] 1.66 KiB {sub/save_list} [built]
[./src/sub/save_list.tsx] 512 bytes {sub/save_list} [built]
    + 42 hidden modules
Child HtmlWebpackCompiler:
     2 assets
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_1 = __child-HtmlWebpackPlugin_1
    [./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html] 549 bytes {HtmlWebpackPlugin_0} [built]
    [./node_modules/html-webpack-plugin/lib/loader.js!./src/sub/save_list.html] 655 bytes {HtmlWebpackPlugin_1} [built]`

please help me.

**Comment by alexander-akait** [comment]
What is the problem? output is good
